README.md

```diff
@@ -79,25 +57,19 @@ source /usr/local/bin/virtualenvwrapper.sh
 
   `cd timewebsite`
 
-1. Create a Python virtualenv to run the project within.
-
-  `mkvirtualenv timewebsite`
-
-  When you return to work on the project again, you will need to re-activate the same virtualenv. This is done by running `workon timewebsite` before attempting to run the project.
-
-1. Install all the Python packages the project uses.
+1. Build/Run the docker container
 
-  `pip install -r requirements.txt`
+  `docker-compose up`
 
 1. Create the local sqlite development database (one time):
 
-  `APP_SETTINGS="config.DevelopmentConfig" DATABASE_URL="sqlite:///timewebsite.db" python db_create.py`
+  `docker exec -it timewebsite_web_1 /bin/bash -c "APP_SETTINGS='config.DevelopmentConfig' DATABASE_URL='sqlite:///timewebsite.db' python db_create.py"`
```

Comment:
You also shouldn't need to to override the 'APP_SETTINGS' envvar since it would already exist